fpx-cli/Cargo.toml

```diff
@@ -69,12 +73,5 @@ tracing-opentelemetry = { version = "0.28" }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 url = { version = "2.5" }
 
-[target.'cfg(windows)'.dependencies]
```

Comment:
Removed this, as the extra feature introduces a bunch of duplicate dependencies. I guess we needed this for some compilation issue on Windows. I think for the moment we should not support Windows until this issue is resolved in `libsql`.